src/compiler/scala/tools/nsc/Reporting.scala

```diff
@@ -115,7 +115,9 @@ trait Reporting extends scala.reflect.internal.Reporting { self: ast.Positions w
       val msg = s"$featureDesc $req be enabled\nby making the implicit value $fqname visible.$explain" replace ("#", construct)
       // don't error on postfix in pre-0.13.18 xsbt/Compat.scala
       def isSbtCompat =
-        (featureName == "postfixOps" && pos.source.path.endsWith("/xsbt/Compat.scala") && Thread.currentThread.getStackTrace.exists(_.getClassName.startsWith("sbt.")))
+        (featureName == "postfixOps" &&
+        (pos.source.path.endsWith("/xsbt/Compat.scala") || pos.source.path.endsWith("""\xsbt\Compat.scala""")) &&
```

Comment:
I thought about converting this to URL etc to make `/` and `\` checking more elegant, but in the end figured doing the String#endsWith twice would be cheaper.

Reply:
You can also use `java.io.File.pathSeparator` in the cross-platform project setup.

Reply:
You mean `java.io.File.separator` I assume. I'm not sure how Cygwin etc interacts on Windows in that regard.

Reply:
I used the manly reflect.io abstraction in combination with java.io. It's part of my regime to get back into shape before the company health expo at the end of October. BTW, last time I watched "Rocky", I noticed that he had six weeks to train. No, not "Rocky and Bullwinkle."